charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
+
+## Objectives
+
+The Unity benchmarking service aims to automate benchmarking for HCA pipelines, enforce standards for consistent reporting for historic comparison and enable the public to participate in suggesting improvements to pipelines by empowering them to implement improvements and test results using the same infrastructure as the HCA data processing pipelines team. The Unity benchmarking service will be used by those making pipelines for the HCA data processing service and will be public, so anyone can benchmark their implemented ideas against HCA pipelines.
+
+## In-scope
+
+* Benchmarking for HCA pipelines will be automated and included in Unity
+* Benchmarking data will be available through Unity but not controllable by the users
+* Benchmarking data can change without notice but will be consistent in format and schema with previous data or notice will be given to users
```

Comment:
Users will be notified if the format or schema for benchmarking data changes? This is the first reference to a _schema_.

Reply:
The UNITY benchmarking service would enforce file formats and file schemas used in the DCP (for instance outputting sequence data in bams and measurements in zarr). Otherwise, the pipelines will not work with the DCP.